packages/components/src/range-control/utils.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * External dependencies
+ */
+import { clamp, noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback, useRef, useEffect, useState } from '@wordpress/element';
+
+/**
+ * A float supported clamp function for a specific value.
+ *
+ * @param {number} value The value to clamp
+ * @param {number} min The minimum value
+ * @param {number} max The maxinum value
+ * @return {number} A (float) number
```

Comment:
Two things:

- Each "column" of the docblock should be aligned, specifically the descriptions in this case.
- `@return` should be separated by a newline (and is not subject to alignment with `@param` parts)

```
 * @param {number} value The value to clamp
 * @param {number} min   The minimum value
 * @param {number} max   The maxinum value
 *
 * @return {number} A (float) number
```